libs/agora/charts/src/lib/median-barchart/median-barchart.component.ts

```diff
@@ -0,0 +1,301 @@
+/* eslint-disable @typescript-eslint/no-this-alias */
+import {
+  AfterViewInit,
+  Component,
+  ElementRef,
+  HostListener,
+  inject,
+  Input,
+  OnChanges,
+  OnDestroy,
+  SimpleChanges,
+  ViewChild,
+  ViewEncapsulation,
+} from '@angular/core';
+import * as d3 from 'd3';
+
+import { HelperService } from '@sagebionetworks/agora/services';
+import { MedianExpression } from '@sagebionetworks/agora/api-client-angular';
+
+@Component({
+  selector: 'agora-median-barchart',
+  standalone: true,
```

Comment:
Note: This option is the default in Angular 19 released in November. We will be able to remove them once we upgrade to that version.

Reply:
I am looking forward to it!